node/buildutils.js

```diff
@@ -57,83 +61,97 @@ exports.getExternals = function () {
     }
 }
 
-var downloadFile = function (url) {
+exports.getExternalsAsync = getExternalsAsync
+
+
+
+/**
+ * @deprecated This method uses library which is not prefered to use on production
+ */
+exports.getExternals = function () {
+    var result = false;
+    getExternalsAsync().then(t => result = true);
+    deasync.loopWhile(function () { return !result });
+    return result;
+}
```

Comment:
This one can be removed.
We may make test function as async and await here: https://github.com/microsoft/azure-pipelines-task-lib/blob/users/ismayilov-ismayil/remove_unsupported_lib/node/make.js#L44